gcloud-java-logging/src/main/java/com/google/gcloud/logging/spi/v2/ConfigServiceV2Api.java

```diff
@@ -0,0 +1,533 @@
+            .newBuilder(payload)
+            .setPageToken((String) token)
+            .build();
+        }
+        @Override
+        public Object extractNextToken(ListSinksResponse payload) {
+          return payload.getNextPageToken();
+        }
+        @Override
+        public Iterable<LogSink> extractResources(ListSinksResponse payload) {
+          return payload.getSinksList();
+        }
+      };
+
+  private static String ALL_SCOPES[] = {
+    "https://www.googleapis.com/auth/logging.read",
+    "https://www.googleapis.com/auth/logging.write",
+    "https://www.googleapis.com/auth/logging.admin"
+  };
+
+  /**
+   * A PathTemplate representing the fully-qualified path to represent
+   * a project_name resource.
+   *
+   * <!-- manual edit -->
+   * <!-- end manual edit -->
+   */
+  private static final PathTemplate PROJECT_NAME_PATH_TEMPLATE =
+      PathTemplate.create("/projects/{project}");
+  /**
+   * A PathTemplate representing the fully-qualified path to represent
+   * a sink_name resource.
+   *
+   * <!-- manual edit -->
+   * <!-- end manual edit -->
+   */
+  private static final PathTemplate SINK_NAME_PATH_TEMPLATE =
+      PathTemplate.create("/projects/{project}/sinks/{sink}");
+
+  // ========
+  // Members
+  // ========
+
+  private final ManagedChannel channel;
+  private final ServiceApiSettings settings;
+
+  // ===============
+  // Factory Methods
+  // ===============
+
+  /**
+   * Constructs an instance of ConfigServiceV2Api with default settings.
+   *
+   * <!-- manual edit -->
+   * <!-- end manual edit -->
+   */
+  public static ConfigServiceV2Api create() throws IOException {
+    return create(new ServiceApiSettings());
+  }
+
+  /**
+   * Constructs an instance of ConfigServiceV2Api, using the given settings. The channels are created based
+   * on the settings passed in, or defaults for any settings that are not set.
+   *
+   * <!-- manual edit -->
+   * <!-- end manual edit -->
+   */
+  public static ConfigServiceV2Api create(ServiceApiSettings settings) throws IOException {
+    return new ConfigServiceV2Api(settings);
+  }
+
+  /**
+   * Constructs an instance of ConfigServiceV2Api, using the given settings. This is protected so that it
+   * easy to make a subclass, but otherwise, the static factory methods should be preferred.
+   *
+   * <!-- manual edit -->
+   * <!-- end manual edit -->
+   */
+  protected ConfigServiceV2Api(ServiceApiSettings settings) throws IOException {
+    ServiceApiSettings internalSettings = ApiUtils.populateSettings(settings,
+        SERVICE_ADDRESS, DEFAULT_SERVICE_PORT, ALL_SCOPES);
+    this.settings = internalSettings;
+    this.channel = internalSettings.getChannel();
+  }
+
+  // ==============================
+  // Resource Name Helper Functions
+  // ==============================
+
+  /**
+   * Creates a string containing the fully-qualified path to represent
+   * a project_name resource.
+   *
+   * <!-- manual edit -->
```

Comment:
Any reason for having this manual edit comment in a javadoc of member variables or methods.

If you really support manual edit of javadoc how do you handle tags?